### PR TITLE
Use .tour property for reporting tournament wins consistently

### PR DESCRIPTION
**Pull request by whalemer**

### CODE REVIEW

**Comment by Slayer95** [comment]
? What's wrong with the current approach?

**Comment by whalemer** [comment]
https://github.com/Zarel/Pokemon-Showdown/blob/5f6af9049f2ec997031783c42c31ae85ed7a32b9/server/rooms.js#L1365-L1370

rooms.js does it differently and it annoys me

It also lets you keep the room as a subroom, so roomstaff can still moderate and the like (which I forgot to implement).

**Comment by whalemer** [comment]
Though on second thoughts I think I'd prefer to do this the other way, and also it's probably a bit less important than I was thinking before.

File: server/tournaments/index.js [hunk 1]
```diff
@@ -991,7 +991,6 @@ class Tournament extends Rooms.RoomGame {
 	 */
 	onBattleWin(room, winnerid) {
 		room.tour = null;
-		room.parent = null;
```
Comment (Zarel) [inline server/tournaments/index.js]: Actually, why did the old code set this to `null`, anyway? It seems unnecessary.

**Comment by Slayer95** [comment]
> Though on second thoughts I think I'd prefer to do this the other way, and also it's probably a bit less important than I was thinking before.

I understand from your words that you'd rather we standardize on using ``.parent``. And I couldn't agree more to that.

**Pull request closed without merging**